### ISSUE TITLE
Make a Script to automate Installation of Project Dependencies for New Contributors

### ISSUE THREAD

**Issue by MaksymKarunos** [ahnaf-zamil/zenora#16]
**Is your feature request related to a problem? Please describe.**
It would be nice to have a script that would install all the dependencies on the new contributor's local machine.
**Describe the solution you'd like**
Anything is fine

**Describe alternatives you've considered**
Make a clear list of instructions and explanations and add it to contributing.md

**Comment by ahnaf-zamil** [comment]
Since this is a complex task, I will work on it. Thanks for raising this issue, I didn't have any idea about even automating it

**Comment by ahnaf-zamil** [comment]
I've finished making in auto project setup and merged the branch with the master. Check out this pull request https://github.com/ahnaf-zamil/zenora/pull/17

When someone downloads the repo for developing, they have to run `python setupproject.py`. This will install all the usage dependencies and the development dependencies. This also installs `pre-commit`, `black` and `flake8`. I've tested this on a separate repo instance and it worked
